packages/app/src/app/index.js

```diff
@@ -66,6 +58,48 @@ const overmind = createOvermind(config, {
   logProxies: true,
 });
 
+if (process.env.NODE_ENV === 'production') {
```

Comment:
We can put this logic in the future probably somewhere in overmind, something more generic. But this is great as a start!